charts/flyte-core/values.yaml

```diff
@@ -821,3 +821,18 @@ sparkoperator:
           - spark.blacklist.enabled: "true"
           - spark.blacklist.timeout: "5m"
           - spark.task.maxfailures: "8"
+
+
+# --------------------------------------------------------
+# Optional Plugins
+# --------------------------------------------------------
+
+# -- Optional: Databricks Plugin allows us to run the spark job on the Databricks platform.
+databricks:
+  enabled: false
+  plugin_config:
+    plugins:
+      databricks:
+        entrypointFile: dbfs:///FileStore/tables/entrypoint.py
+        # Databricks account
+        databricksInstance: dbc-a53b7a3c-614c
```

Comment:
How do we deal with multiple databricks environments? I guess you are assuming one environment of dbx. Is there an ability to overwrite this value at workflow submission?

Reply:
I though you only have one dbx account. Let me add it to the dbx config on flytekit-side